plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHiveFileFormats.java

```diff
@@ -640,7 +640,7 @@ public static FileSplit createTestFileHive(
                 () -> {});
 
         try {
-            serializer.initialize(new Configuration(false), tableProperties);
+            serializer.initialize(newEmptyConfiguration(), tableProperties);
```

Comment:
I see you did not change calls to `new JobConf()` (parameterless). Is that because those are in tests only?
Should those be disallowed in the production code by modernized?

Reply:
> I see you did not change calls to `new JobConf()` (parameterless). Is that because those are in tests only?

because i eventually forgot to do that.